package.json

```diff
@@ -124,6 +130,7 @@
     "scroll-to-element": "^2.0.0",
     "sprintf-js": "1.0.3",
     "style-loader": "^3.0.0",
+    "ts-node": "^10.0.0",
```

Comment:
I don't know what magic this does, but this is how everything magically runs with `node`

Reply:
I assume webpack uses it if it's available to read TS